sechub-pds-tools/src/main/java/com/mercedesbenz/sechub/pds/tools/PDSToolsCLI.java

```diff
@@ -0,0 +1,79 @@
+// SPDX-License-Identifier: MIT
+package com.mercedesbenz.sechub.pds.tools;
+
+import java.io.File;
+
+import com.mercedesbenz.sechub.pds.tools.generator.PDSSolutionTestFilesGenerator;
+import com.mercedesbenz.sechub.pds.tools.handler.ConsoleHandler;
+import com.mercedesbenz.sechub.pds.tools.handler.ExitHandler;
+import com.mercedesbenz.sechub.pds.tools.handler.PrintStreamConsoleHandler;
+import com.mercedesbenz.sechub.pds.tools.handler.SystemExitHandler;
+
+public class PDSToolsCLI {
+
+    ConsoleHandler consoleHandler;
+    ExitHandler exitHandler;
+
+    public PDSToolsCLI() {
+        consoleHandler = new PrintStreamConsoleHandler();
+        exitHandler = new SystemExitHandler();
+    }
+
+    public static void main(String[] args) throws Exception {
+        PDSToolsCLI cli = new PDSToolsCLI();
+        cli.start(args);
+    }
+
+    void start(String... args) throws Exception {
+        if (args.length == 0) {
+            showHelpAndExit("Arguments missing", 1);
+        }
+        String command = args[0];
+        switch (command) {
+
+        case "--help":
+            showHelp();
+            exitHandler.exit(0);
+        case "--generate":
+            PDSSolutionTestFilesGenerator generator = new PDSSolutionTestFilesGenerator();
+            generator.setOutputHandler(consoleHandler);
+
+            if (args.length < 3 || args.length > 4) {
+                showHelpAndExit("Generate command needs 2 additional parameters: 1. config file path, 2.scan type", 3);
+            }
+            String pathToConfigFile = args[1];
+            String scanType = args[2];
```

Comment:
It would be nice to verify the scan type. Is there an enum for the scan type which can be used to check the value?

Reply:
Scan type verification is done by the called generator inside.